typescript/src/feast/pubsub/google.ts

```diff
@@ -0,0 +1,6 @@
+import {APIGatewayProxyEvent, APIGatewayProxyResult} from "aws-lambda";
+
+export async function handler(request: APIGatewayProxyEvent): Promise<APIGatewayProxyResult>  {
+    console.log(`${JSON.stringify(request)}`)
```

Comment:
hardly worth adding now, but it makes it slightly easier to review if there's a comment explaining the body
```suggestion
    // placeholder for new lambda
    console.log(`${JSON.stringify(request)}`)
```